src/osd/ECBackend.cc

```diff
@@ -180,6 +180,7 @@ ECBackend::ECBackend(
   : PGBackend(pg, store, coll, ch),
     cct(cct),
     ec_impl(ec_impl),
+    extent_cache(),
```

Comment:
You don't need to do this explicitly, please don't, it's inconsistent with the rest of the code.